app/auth/models.py

```diff
@@ -152,3 +153,42 @@ def create_user(self, email, full_name, user_name, password):
             return response.json()
         else:
             return None
+
+
+class FileData(object):
```

Comment:
@rufuspollock This was not implemented before. Previously `finalize` API had only one method call from `Bitstore` and get one URL and return back. Right now we have to build a big json to return back to cli. 
That is why i separated the code from controller, so that it can be testable.